blogposts/2021/release-3.31.md

```diff
@@ -0,0 +1,88 @@
+---
+title: "Sourcegraph 3.31 release"
+publishDate: 2021-08-20T10:00-07:00
+description: "Sourcegraph 3.31 introduces UPDATE"
+tags: [blog, release]
+slug: "release/3.31"
+published: false
+heroImage: 
+socialImage: 
+changelogItems:
+  - description: "Backend Code Insights GraphQL queries now support arguments `includeRepoRegex` and `excludeRepoRegex` to filter on repository names."
+    url: https://github.com/sourcegraph/sourcegraph/pull/23256
+    category: Code Insights
+  - description: Operator documentation has been added to the Search Reference sidebar section.
+    url: https://github.com/sourcegraph/sourcegraph/pull/23116
+    category: Search
+  - description: Syntax highlighting support for the Cue language.
+    url: https://cuelang.org
+    category: Search
+  - description: Reintroduced a revised version of the Search Types sidebar section.
```

Comment:
@iskyOS - can you add what the benefit is/how someone would use the Search Types sidebar section?

Reply:
Yep- added.